docs/src/recipe-basemap-postgis.md

```diff
@@ -0,0 +1,146 @@
+# Setting up a Basemap and Overlaying Points from PostGIS
+
+You commonly have some semi-proprietary datasource which you want to overlay onto another.
+This guide shows how to generate a basemap using [Planetiler](https://github.com/onthegomap/planetiler/) from [OSM](https://osm.org) and overlay custom points from a [PostGIS](https://postgis.net/) database.
+See here for a discussion on the [pros and cons of this/alternative data sources](sources-tiles.md).
+
+## Prerequisites
+
+We expect you have the following already installed:
+
+- [Docker](https://docker.io)
+- [Martin binary](installation.md)
+
+## Generate an MBTiles basemap with Planetiler
+
+There are multiple ways to generate a tile archive.
+For semi-static tile archives, we think using [Planetiler](https://github.com/onthegomap/planetiler/) to build MBtiles archives using  [OpenMapTiles](https://openmaptiles.org/) is a good starting point.
+
+<details><summary>🤔 <i>Why do I need a tool to convert OSM to mbtiles in the first place?</i> (click to expand)</summary>
+
+The reason you need a tool to build vector tilesets from OpenStreetMap is that the data in OpenStreetMap is
+
+- not following a specific schema,
+- nor pre-tiled into `x`/`y`/`z` chunks.
+
+</details>
+
+<details><summary>🤔 <i>What is up with MBtiles and OpenMapTiles</i> (click to expand)</summary>
+
+Good question.
+
+MBtiles is the archive format. Think of a sqlite database storing the data you need a chunk (`x`/`y`/`z`) of the world.
+See our comparison [pmtiles vs. mbtiles](sources-tiles.md) for discussions on the pros and cons of this/alternative formats.
+
+But how does the data in the archive look like?
+This is where the vector tile schema comes in:
+[OpenMapTiles](https://openmaptiles.org/) defines which layers are included in the served data and how they are aggregated.
+[OpenMapTiles](https://openmaptiles.org/) does have an attribution requirement. You will need to add `© MapTiler` at the bottom of your map.
```

Comment:
I do not believe this is correct.  Because OMT is under CC-BY license, you have to give credit to OpenMapTiles, but not to the owner of OMT.

Reply:
Will need to check, but at least OSM Liberty (which I think know is based on openmaptiles) credits this.

Maybe they do relinquish the copyright internally. Will need to check.

Reply:
See https://github.com/openmaptiles/openmaptiles?tab=readme-ov-file#license :

> Products or services using maps derived from OpenMapTiles schema need to visibly credit "OpenMapTiles.org" or reference "OpenMapTiles" with a link to https://openmaptiles.org/. Exceptions to attribution requirement can be granted on request.